compiler/compiler-core.cpp

```diff
@@ -335,6 +336,16 @@ void CompilerCore::require_function(FunctionPtr function, DataStream<FunctionPtr
   }
 }
 
+void CompilerCore::parse_tracked_builtins(const std::string& file_name) {
+  std::ifstream tracked_builtins_file(file_name);
+  kphp_error(tracked_builtins_file.is_open(), "Cannot open file with list of tracked builtins");
+
+  std::string function_name;
+  while (tracked_builtins_file >> function_name) {
+    tracked_builtins.emplace(std::move(function_name));
+  }
+}
+
```

Comment:
Is file convenient?
How to add it in CI/CD apps? Aren't env vars more convenient for that?

Reply:
Make it as option and env